client/src/components/Navbar/Navbar.tsx

```diff
@@ -110,18 +115,36 @@ const Navbar: React.FC = () => {
     handleClose();
     logout();
   };
+  const [profile, setProfile] = useState<Profile>();
 
   const renderMenuItems = () => {
-    // TODO: conditionally render based on profile type
+    // Add a different view if the profile is null? Have a link suggesting them to create a profile?
```

Comment:
When a new user signs up we create a profile for them too - I don't think think we need to worry about redirecting them to create a profile (we can remove this comment here)